test/taskrun_test.go

```diff
@@ -122,8 +123,10 @@ func TestTaskRunFailure(t *testing.T) {
 	}
 
 	releaseAnnotation, ok := taskrun.Annotations[pod.ReleaseAnnotation]
-	if !ok || releaseAnnotation != "devel" {
-		t.Fatalf("expected Taskrun to be annotated with %s=devel, got %s=%s", pod.ReleaseAnnotation, pod.ReleaseAnnotation, releaseAnnotation)
+	// Nightly release tag looks like v20210323-0593c7bef3
+	nightlyReleaseRegexp := regexp.MustCompile("^v[0-9]{8}-[0-9a-z]{10}$")
```

Comment:
this is interesting, date followed by commit hash?

Reply:
@pritidesai fyi https://github.com/tektoncd/plumbing/blob/4823ac9a4413ec3d716a9c2ccd2f9d61f775149b/tekton/cronjobs/bases/release/trigger-with-uuid.yaml#L41